scheduler/core/programprovider/ocs/ocsprogramprovider.py

```diff
@@ -44,20 +44,15 @@
 DEFAULT_PROGRAM_ID_PATH = Path(ROOT_DIR) / 'scheduler' / 'data' / 'program_ids.txt'
 
 
-def ocs_program_data(program_list: Optional[bytes] = None) -> Iterable[dict]:
+def ocs_program_data(program_list: Optional[List[str]] = None) -> Iterable[dict]:
```

Comment:
Are we just passing a list of strings,  would this not break the file using in other stuff like greedymax script and fill_redis script?

Reply:
Yes, you are right, it will break the other scripts, let me see what I can do to have both input types in the ocs_program_data.